archive/c/commodore-basic/longest-word.bas

```diff
@@ -0,0 +1,30 @@
+5 REM Input string
+10 GOSUB 1000
+20 IF S$ = "" OR S$ = CHR$(13) THEN GOTO 160
+25 REM Keep track of length of longest word
+30 LW% = 0: REM longest word length so far
+40 LC% = 0: REM current word length
+50 FOR K = 1 TO LEN(S$)
+60     A$ = MID$(S$, K, 1)
+70     IF A$ = CHR$(9) OR A$ = CHR$(10) OR A$ = CHR$(13) THEN GOTO 120
+80     IF A$ = " " THEN GOTO 120: REM can't put on same line, too long
+90     LC% = LC% + 1
+100     IF LC% > LW% THEN LW% = LC%
```

Comment:
The increase in digit size here makes for weird indentation. I feel like this would bother me enough to make all the labels three or four digits by default.

Reply:
Yeah, that does look weird. I just go in the habit of indenting by 4 for after `"<line_number> "` for `FOR` loops, and I didn't pay attention to how it looked.